(Also A) First Time Speaker blog

nickloestartup opened the pull request:
### (Also A) First Time Speaker

### My experience speaking at an Agile conference for the first time

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

hxmattobee commented on _posts/2017-06-29-also-a-first-time-speaker.md:
```diff
@@ -0,0 +1,89 @@
+It’s the first time I’ve spoken at an Agile conference. I was determined to do it this year having been to conferences over the last couple of years and thinking “we do that already” or “We’ve already learnt that and implement that better”. It was time I got out there and started to talk about some of the great stuff we do in the Web Team at Holiday Extras.
+
+Before I set about writing this blog, I re-read this excellent post Liz wrote about her experience last year being a [First Time Speaker](http://tech.holidayextras.co.uk/speaking/2016/06/07/First-time-speaker/). A lot of that rang true for me:
+* Not feeling prepared
```
Need to add a blank line before the first list item, otherwise it won't be formatted properly as a bulleted list.

hxmattobee commented on _posts/2017-06-29-also-a-first-time-speaker.md:
```diff
@@ -0,0 +1,89 @@
+
+I started to put the talk together a couple of months before the conference. Start early, be ready early.
+
+As the weeks went on, I managed to progress the presentation. It took a while to be happy with a consistent design, but once I got there I was able to transfer over a lot of the presentation from our Conference Day. I’d topped the presentation with the introductory information needed for Holiday Extras and I knew how I wanted to end it.
```
I wonder if "managed to make progress on the presentation" would sound better than "managed to progress the presentation"?

hxmattobee commented on _posts/2017-06-29-also-a-first-time-speaker.md:
```diff
@@ -0,0 +1,89 @@
+During lunch, I went in the room I’d be presenting in. I looked at the HDMI cable and panicked. There was no way that was fitting in my laptop! I recalled the speakers pack had spoken about bringing the the right adaptor. What to do now? In blind panic I tried the cable anyway. Oh, it does fit - panic over! So, I’d plugged in my laptop and I started to see if everything was OK, while people start ...
+
+OK, everything seems to work fine. I can wander around, my clicker works, the slides look great up on the big screen. Let’s have some lunch:
+![agile-in-the-city-food]({{ site.baseurl }}/assets/img/agile-in-the-city-food.png)
```
This should be .jpg not .png

HanCork89 replied:
Switch to present tense again here but it makes more sense here :)

hxmattobee commented on _posts/2017-06-29-also-a-first-time-speaker.md:
```diff
@@ -0,0 +1,89 @@
+
+## It’s actually happening
+
+So the time came, I plugged in and got going. I cracked my first joke in the first minute or so and the audience seemed to appreciate it and were engaged. My technical hitch on the video was navigated well and people seemed to enjoy seeing around our campus here at Holiday Extras. The video ends, a list of my most recently watched videos appears in all it’s glory. Some I’d watched for work, some  ...
```
Change "it’s glory" to "its glory".

HanCork89 commented on _posts/2017-06-29-also-a-first-time-speaker.md:
```diff
@@ -0,0 +1,89 @@
+I recently spoke at [Agile in the City](http://agileinthecity.net/2017/london/). Honest! Here look at the programme, it’s me!
+![agile-in-the-city-programme]({{ site.baseurl }}/assets/img/agile-in-the-city-programme.png)
+
+It’s the first time I’ve spoken at an Agile conference. I was determined to do it this year having been to conferences over the last couple of years and thinking “we do that already” or “We’ve already learnt that and implement that better”. It was time I got out there and started to talk about some of the great stuff we do in the Web Team at Holiday Extras.
```
"we do..." "...We've..." - should probably be consistent in either capitalising or not

HanCork89 commented on _posts/2017-06-29-also-a-first-time-speaker.md:
```diff
@@ -0,0 +1,89 @@
+
+The talk was a case study, and one I’d already partially delivered at the Holiday Extras Conference Day (an internal Tech conference). That talk had gone OK and given it was a case study, I felt confident I could do the subject matter justice - after all it was just talking about my own experiences.
+
+Even the week before, weekend before, day before and on the day, I’m still convincing myself that I’ll be fine. I know the subject well, I can talk about it without any problems right?
```
You switched to present tense here with "I'm",  "I'll" and "know" - maybe "I was", "I would" and "knew" would be better?

HanCork89 commented:
I've added a couple of comment but they're only small points so I'd be happy with this to go out as is. Nice post, well done!

nickloestartup commented:
Thanks @HanCork89 @hxmattobee. I've addressed both your comments. I've also added a hero image and promoted the blog, so I'll get someone to check that before I publish as I know you're both off.

nickloestartup commented:
@LouisPryer I'd like to get this published. I don't think your PR has been reviewed / approved. So I'm assuming I need to remove the 'promoted' from the current promoted blog like I have? Does this look good to go and can be merged? Thanks

nickloestartup commented:
This has been moved to https://github.com/holidayextras/holidayextras.github.com/pull/178, so closing this one
